src/mms_client/utils/web.py

```diff
@@ -132,6 +134,7 @@ def __init__(
         interface: Interface,
         adapter: Pkcs12Adapter,
         logger: Logger,
+        auditer: Optional[AuditPlugin] = None,
```

Comment:
I think this should just be an Optional[Plugin] type. Let the people decide for themselves which plugin they want.

Reply:
That's a good point.